### PR TITLE
use pk instead of id in clear expired tokens

### PR DESCRIPTION
**Pull request by sahama**
Fixes #

## Checklist

- [x] PR only contains one change (considered splitting up PR)
- [ ] unit-test added
- [ ] documentation updated
- [ ] `CHANGELOG.md` updated (only for user relevant changes)
- [x] author name in `AUTHORS`

### CODE REVIEW

File: oauth2_provider/models.py [hunk 1]
```diff
@@ -532,7 +532,7 @@ def revoke(self):
             self = list(token)[0]
 
             with suppress(access_token_model.DoesNotExist):
-                access_token_model.objects.get(id=self.access_token_id).revoke()
+                access_token_model.objects.get(pk=self.access_token_id).revoke()
```
Comment (merito) [inline oauth2_provider/models.py:540]: I'm not sure if it will work

**Comment by dopry** [comment]
@sahama  are you willing to complete the work on this PR or should we close it?

File: oauth2_provider/models.py (hunk 1 above)
Comment (dopry) [reply oauth2_provider/models.py:540]: @merito  why not?

**Comment by dopry** [comment]
@n2ygk  I implemented tests for this branch. I'd appreciate it if you could give it a double check. I feel adding the pk support is correct... however, I am concerned about adding one more thing we need to support and test against and one more edge case that can go wrong.

**Review by n2ygk — APPROVED** [review]
Using pk sounds like the right thing to do
